jicofo & jvb: Fix OCTO + SCTP behaviour

spijet opened the pull request:
Hello everyone!

As a follow-up to #1787, here's the PR that:
1. Removes the bogus `cont-init` check;
2. Enables SCTP by default in Jicofo config template.

I'll leave the PR as a draft for now, until @saghul confirms that all that needed to be done is done. :)

saghul commented on jicofo/rootfs/defaults/jicofo.conf:
```diff
@@ -4,10 +4,10 @@
 {{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "0" | toBool -}}
 {{ $ENABLE_OCTO := .Env.ENABLE_OCTO | default "0" | toBool -}}
-{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default (.Env.ENABLE_SCTP | default "0") | toBool -}}
+{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default (.Env.ENABLE_SCTP | default "1") | toBool -}}
```
I think you can simplify this by defaulting to $ENABLE_SCTP

spijet replied:
Is it better now? :)

saghul commented:
@damencho can you PTAL too?

damencho commented:
> @damencho can you PTAL too?

I asked @bgrozev and he said it is fine :)

spijet commented:
I updated the config template according to @saghul's comment and switched this PR from draft to open. Please let me know if there's anything else that needs to be added. :)

saghul commented on jicofo/rootfs/defaults/jicofo.conf:
```diff
@@ -4,10 +4,10 @@
 {{ $JICOFO_AUTH_TYPE := .Env.JICOFO_AUTH_TYPE | default $AUTH_TYPE -}}
 {{ $JICOFO_AUTH_LIFETIME := .Env.JICOFO_AUTH_LIFETIME | default "24 hours" -}}
-{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "0" | toBool -}}
+{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "1" | toBool -}}
```
Upon further inspection I think we should keep this defaulting to disabled, for the time being. With the rest of the changes inplace you'll still be able to enable OCTO over SCTP though.

spijet replied:
Done in b2ecc27. :)

spijet commented:
Yay! ❤️
